apps/meteor/app/lib/server/functions/addOAuthService.ts

```diff
@@ -9,27 +9,31 @@ export async function addOAuthService(name: string, values: { [k: string]: strin
 	name = capitalize(name);
 	await settingsRegistry.add(`Accounts_OAuth_Custom-${name}`, values.enabled || false, {
 		type: 'boolean',
+		public: false,
```

Comment:
there is no need to say `public: false` since all settings have this by default, see:
https://github.com/RocketChat/Rocket.Chat/blob/d34ce1569bfa01b99477751b07c5726a9d511e73/apps/meteor/app/settings/server/functions/getSettingDefaults.ts#L35